testdata/p4_16_samples_outputs/issue561-frontend.p4

```diff
@@ -16,9 +16,8 @@ package top(ct _ct);
 control c(out bit<32> x) {
     @name("c.u") U u_0;
     @name("c.u2") U[2] u2_0;
-    @name("c.b") bool b_0;
     apply {
-        b_0 = u_0.isValid();
+        u_0.isValid();
```

Comment:
Please file an issue so that we add code to detect such method calls that have no effect and remove them too. It's funny that we did that in some places for assignments, but not just for method calls.

Reply:
I've opened up a new issue: https://github.com/p4lang/p4c/issues/2628